cmd/cli/cmdimport.go

```diff
@@ -62,35 +61,35 @@ func ImportKey(filename, keyPassword string, keyPasswordFd int) {
 		keyPassword = strings.Trim(string(d), "\n\r")
 	}
 
-	fps, _ := remote_signer.GetFingerPrintsFromKey(string(data))
+	fps, _ := tools.GetFingerPrintsFromKey(string(data))
```

Comment:
No error handling. ;)

Reply:
Don't need. The key is parsed before that.

Reply:
Done!